### ISSUE TITLE
Need a "reset" for when it gets "corrupted."

### ISSUE THREAD

**Issue by inspectionsbybob** [5axes/UniversalCuraSettings#30]
After playing with UCS for a while I got to a configuration that was printing seriously ugly. Under Extrusion like crazy.  There was no way to "back out" or "undo". Tried pulling in a "default" profile to reset my custom profile but it still was miss-behaving.

I switched slicers to double check it wasn't wet filament or something on the printer, and the other slicer worked fine.  Went through ALL the settings AGAIN and couldn't find anything.  Had this happen before without UCS and had to do a clean re-install, ie delete the WinData stuff as well and rebuild my profiles from scratch.

**Need a way to get the WHOLE application back to a known state, ie a profile "RESET".**

just deleting a profile doesn't seem to delete/reset everything. Some associated things in the printer and system options seem to stay, and I think somewhere in there is where my system got corrupted.

**Comment by 5axes** [comment]
Yes you're right , that's on my point of view the limit of the profile technics. As you don't store everything in the profile sometimes it could be difficult to return to the orginal state. One work around is to use the function of this plugin call "Export Current Profile"  you can save the current profile into a CSV file. And the reset the values by using the second function "Merge a profile"
![image](https://user-images.githubusercontent.com/11015345/160433953-cb87af8c-a99d-4767-89f5-8b26b3e9fcf1.png)
